src/WebAppDIRAC/WebApp/static/DIRAC/JobMonitor/classes/JobMonitor.js

```diff
@@ -727,7 +683,7 @@ Ext.define("DIRAC.JobMonitor.classes.JobMonitor", {
       mode: "local",
       store: new Ext.data.ArrayStore({
         fields: ["category"],
-        data: [["Status"], ["Site"], ["Minor Status"], ["Application Status"], ["Owner"], ["Owner Group"], ["Job Group"], ["Job Type"]],
+        data: [["Status"], ["Site"], ["Minor Status"], ["Application Status"], ["Owner"], ["VO"], ["Job Group"], ["Job Type"]],
```

Comment:
Should this still have "Owner Group" ? I don't know what this does at all.

Reply:
Done.